.github/workflows/build_test.yml

```
@@ -103,7 +155,7 @@ jobs:
 
   controller-e2etest:
     name: E2E Controller
-    runs-on: ubuntu-latest
+    runs-on: ${{ needs.authorize.outputs.runner || 'ubuntu-latest' }}
```

actionlint (1.7.7)

property "authorize" is not defined in object type {lint-test: {outputs: {clusterprefix: string; pr_merge_commit: string; version: string}; result: string}}

(expression)